### DISCUSSION TITLE
SITS-1.4.2-1 released: improved methods for time series analysis of data available in MPC

### DISCUSSION THREAD

**Discussion by gilbertocamara** [microsoft/PlanetaryComputer#293] [General]
Dear colleagues

We would like to announce the release of version 1.4.2-1 of the SITS R package. SITS provides an end-to-end environment for land use and land cover classification using big EO data. Users are supported by an on-line book (https://e-sensing.github.io/sitsbook/). The software has matured and reached TRL 9 status; it is being used operationally for LUCC mapping in large areas of Brazil.  

We fully support the Sentinel-2 and Landsat collections in MPC. In the new version, initial support for Sentinel-1 is available. You can use SITS in MPC using the R notebooks available in the MPC hub. 

Improvements of this version include: 
(a) Support for vector data cubes, including visualisation;
(b) Object-based time series analysis using spatio-temporal segmentation.
(c) Improved support for GPU usage when running deep learning algorithms. Performance of GPU of DL methods is equivalent to that of PyTorch;

SITS benefits from generous support from Microsoft for its development and deployment. 

Best regards
Gilberto Camara
Senior Researcher
National Institute for Space Research (INPE), Brazil
https://gilbertocamara.org/

**Comment by TomAugspurger** [comment]
Congrats on the release!
